app/views/articles/_collection.html.erb

```diff
@@ -2,12 +2,18 @@
 <% slugs = articles.pluck(:slug) %>
 
 <% if collection && collection_size > 1 %>
+  <% size_string = "#{collection_size} Part Series" %>
   <div class="article-collection-wrapper article-collection-wrapper-<%= position %>">
-    <% if collection.slug.present? %>
-      <p><%= collection.slug %> (<%= collection_size %> Part Series)</p>
-    <% else %>
-      <p><%= collection_size %> Part Series</p>
-    <% end %>
+    <p>
+      <a href="<%= collection.path %>">
```

Comment:
**nitpick (non-blocking):** We're using this in two places, maybe we can wrap it up in a helper, which would also remove some local variables (`size_string`) from views. Something like:

```ruby
def collection_link(collection, options = {})
  collection_size = ...
  size_string = ...
  body = if collection.slug.present? ...

  link_to body, collection.path, options
end
```

Reply:
The way you suggest looks cleaner, I'm pushing a commit with the change!